### PR TITLE
Create necromancer.lua

### PR DESCRIPTION
**Pull request by MisterClems**
French translation : Necromancer

### CODE REVIEW

File: lua/terrortown/lang/french/necromancer.lua [hunk 1]
```diff
@@ -0,0 +1,36 @@
+L = LANG.GetLanguageTableReference("french")
+
+-- GENERAL ROLE LANGUAGE STRINGS
+L[NECROMANCER.name] = "Nécromancien"
+L[NECROMANCER.defaultTeam] = "Team Nécromanciens"
+L["hilite_win_" .. NECROMANCER.defaultTeam] = "LE NÉCROMANCIEN A GAGNÉ"
+L["win_" .. NECROMANCER.defaultTeam] = "Le Nécromancien a gagné!"
+L["info_popup_" .. NECROMANCER.name] = [[Maintenant, c'est à votre tour ! Essayez de faire combattre des joueurs morts à vos côtés!]]
+L["body_found_" .. NECROMANCER.abbr] = "C'était un Nécromancien!"
+L["search_role_" .. NECROMANCER.abbr] = Cette personne était un Nécromancien!"
```
Comment (Histalek) [inline lua/terrortown/lang/french/necromancer.lua]:
```suggestion
L["search_role_" .. NECROMANCER.abbr] = "Cette personne était un Nécromancien!"
```

**Review by Histalek — CHANGES_REQUESTED** [review]
just a missing `"`

**Comment by MisterClems** [comment]
fixed

**Review by Histalek — APPROVED** [review]

File: lua/terrortown/lang/french/necromancer.lua [hunk 2]
```diff
@@ -0,0 +1,36 @@
+L = LANG.GetLanguageTableReference("french")
```
Comment (Alf21) [inline lua/terrortown/lang/french/necromancer.lua]:
```suggestion
local L = LANG.GetLanguageTableReference("french")
```

**Review by Alf21 — CHANGES_REQUESTED** [review]

**Comment by MisterClems** [comment]
fixed

**Review by Alf21 — APPROVED** [review]

**Pull request merged**